ocs_ci/framework/conf/default_config.yaml

```diff
@@ -138,3 +140,8 @@ UPGRADE:
 # or ping in ocs infra chat room.
 AUTH:
     test_quay_auth: test_secret
+
+# Defines the configuration of Independent Ceph Cluster
+INDEPENDENT_MODE:
+  admin_keyring:
```

Comment:
Does this info warrant an entirely new top level section for our config? It seems like it would be easy enough to add under the DEPLOYMENT section as `independent_mode_keyring` or something similar.

Reply:
We actually have couple other variables that are specific to this mode, I think it makes sense to move them to its own section (ip/user/pass/pool config etc)